test/test_retriever.py

```diff
@@ -414,8 +415,8 @@ def test_add_dialect():
     table['dialect']['dummy_key'] = 'dummy_value'
 
     result = {}
-    result['cleanup'] = 'Cleanup(correct_invalid_value, missing_values=\x00)'
-    result['delimiter'] = "'\t'"
+    result['cleanup'] = Cleanup(correct_invalid_value, nulls='\x00')
```

Comment:
I think we should keep missing_values? Unless I am missing something

Reply:
Yeah this should be missing_values. That was my bad!